### PR TITLE
Update to reflect status of Clang 4 and 5

### PR DESCRIPTION
**Pull request by acgetchell**
https://github.com/travis-ci/travis-ci/issues/8346

### CODE REVIEW

**Comment by AraHaan** [comment]
Maybe the issue is because gcc's standard libary directories are somehow first on the list and so clang confuses the two. Maybe during the builds modify it to temporarily uninstall gcc or upgrade it?

**Comment by acgetchell** [comment]
The issue seems to be that the clang-4 and clang-5 packages aren't found at all.

**Comment by AraHaan** [comment]
oh, I see.

**Comment by plaindocs** [comment]
I've merged #1432 into #1410 - when is merged to master does that fix this issue? Or is there more outstanding I need to look into?

File: _includes/c11-cpp11-and-beyond-and-toolchains.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ matrix:
       env:
         - MATRIX_EVAL="CC=clang-3.9 && CXX=clang++-3.9"
 
-    # works on Trusty
+    # Currently doesn't work on Trusty
```
Comment (AraHaan) [inline _includes/c11-cpp11-and-beyond-and-toolchains.md]: This one seems to work except for the issue with needing to uninstall gcc or upgrading gcc as well as clang only see's gcc's header files first and so it uses them to compile code. (Which is what makes this fail for anything C++ 14 or newer.

File: _includes/c11-cpp11-and-beyond-and-toolchains.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ matrix:
       env:
         - MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
 
-    # works on Trusty
+    # Currently doesn't work on Trusty
```
Comment (AraHaan) [inline _includes/c11-cpp11-and-beyond-and-toolchains.md]: This does not work still if I can see, Will restart a build later to double check.

File: _includes/c11-cpp11-and-beyond-and-toolchains.md (hunk 1 above)
Comment (Mizux) [reply _includes/c11-cpp11-and-beyond-and-toolchains.md]: seems related to https://github.com/travis-ci/travis-ci/issues/9550#issuecomment-433030978
i.e. just need to add CPATH to MATRIX_EVAL should be sufficient IMHO

Comment (Mizux) [reply _includes/c11-cpp11-and-beyond-and-toolchains.md]: working example: https://github.com/abseil/abseil-cpp/pull/197/files#diff-354f30a63fb0907d4ad57269548329e3

**Comment by plaindocs** [comment]
Hey folks, I'm not quite sure what the status of this is so I'm closing. Please reopen with more info if still valid.

**Pull request closed without merging**